code/3_train.py

```diff
@@ -1,21 +1,6 @@
-# DeepSlide
-# Jason Wei, Behnaz Abdollahi, Saeed Hassanpour
-
-# Training the resnet
-
-from utils_model import train_resnet
-
-if __name__ == '__main__':
-    train_resnet(train_folder=config.train_folder,
-                 num_epochs=config.num_epochs,
-                 num_layers=config.num_layers,
-                 learning_rate=config.learning_rate,
-                 batch_size=config.batch_size,
-                 weight_decay=config.weight_decay,
-                 learning_rate_decay=config.learning_rate_decay,
-                 resume_checkpoint=config.resume_checkpoint,
-                 resume_checkpoint_path=config.resume_checkpoint_path,
-                 save_interval=config.save_interval,
-                 checkpoints_folder=config.checkpoints_folder,
-                 pretrain=config.pretrain,
-                 log_csv=config.log_csv)
+from utils_model import train_resnet
+
+# Training the ResNet.
+print("+++++ Running 3_train.py +++++")
+train_resnet()

```

Comment:
I don't think that these parameters should be taken away. 

It's nice to see what the exact inputs are for this function.

WDYT?

Reply:
That was something I wasn't entirely sure about.  I agree that it's nice to see what's being passed in.

Reply:
Parameters added back in.  Now the `utils_*.py` files don't have access to the `config.py` file.